ext/zip/php_zip.stub.php

```diff
@@ -721,6 +720,27 @@ class ZipArchive implements Countable
      */
     public const LIBZIP_VERSION = UNKNOWN;
 
+    /* since 1.10.1 */
+#ifdef ZIP_LENGTH_TO_END
+    /**
+     * @var int
+     * @cvalue ZIP_LENGTH_TO_END
+     */
+    public const LENGTH_TO_END = UNKNOWN;
```

Comment:
you can now declare the constant types :) P.S.: Would you be OK to convert all ZipArchive class constants to typed?

Reply:
I prefer to not, as same code is also used for older PHP versions, and don't want to much diff.

Reply:
Btw, compat macro works, so done

Also in pecl https://github.com/pierrejoye/php_zip/commit/f8abd19d673e09a8f008abb8ae8623973c3a1833